File: app/src/main/java/org/thoughtcrime/securesms/util/AccessibilityUtil.java
package org.thoughtcrime.securesms.util;

import android.content.Context;
import android.provider.Settings;

public final class AccessibilityUtil {

  private AccessibilityUtil() {
  }

  public static boolean areAnimationsDisabled(Context context) {
    return Settings.Global.getFloat(context.getContentResolver(), Settings.Global.ANIMATOR_DURATION_SCALE, 1) == 0f;
  }
}

File: app/src/main/java/org/thoughtcrime/securesms/util/AccessibilityUtil.kt
package org.thoughtcrime.securesms.util

import android.content.Context
import android.provider.Settings

object AccessibilityUtil {
  @JvmStatic
  fun areAnimationsDisabled(context: Context): Boolean {
    return Settings.Global.getFloat(context.contentResolver, Settings.Global.ANIMATOR_DURATION_SCALE, 1f) == 0f
  }
}
